### PR TITLE
bugfix/createDevon4jProject

### PR DESCRIPTION
**Pull request by denise-khuu**
createDevon4jProject("com.example.friendexample")

should be a valid argument and should create a folder named friendexample.

### CODE REVIEW

**Review by GuentherJulian — APPROVED** [review]

File: documentation/Development.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 In this section we will describe how you can implement your own tutorials.
 Learn more about the file system and where you can add your commands in 
-https://github.com/devonfw-forge/tutorials/wiki/Structure
+https://github.com/devonfw-forge/tutorial-compiler/wiki/Structure
```
Comment (SchettlerKoehler) [inline documentation/Development.md:5]: https://github.com/devonfw-tutorials/tutorial-compiler/wiki/Structure ?

File: documentation/Functions.md [hunk 2]
```diff
@@ -143,9 +143,10 @@ cobiGenJava("path/to/java/file/MyEntity.java",[1,3,5,6,8])
 
 ### createDevon4jProject
 #### parameter 
-1. The project name
+1. The base package name
 #### example 
-createDevon4jProject("cobigenexample")
+createDevon4jProject("com.example.application.cobigenexample")
```
Comment (SchettlerKoehler) [inline documentation/Functions.md:148]: Add better example like in https://github.com/devonfw/devon4j/issues/388

File: documentation/Development.md (hunk 1 above)
Comment (denise-khuu) [reply documentation/Development.md:5]: that's the page, where we explain the file structure and 'working directory' and 'workspace directory'

Comment (SchettlerKoehler) [reply documentation/Development.md:5]: Yes, but the link is wrong. It should be devonfw-tutorials not devonfw-forge

**Review by SchettlerKoehler — APPROVED** [review]

**Pull request merged**